Assets/Scripts/Buddy/BuddyShaper.cs

```diff
@@ -53,28 +60,46 @@ void Start()
 		}
 	}
 
-	void RandomizeBuddy()
+	void Update()
 	{
-		for( int i = 0; i < _numBlendShapes; i++ )
+		if( _debugRandomizer != _prevDebugRandomizer )
 		{
-			_skinnedMeshRend.SetBlendShapeWeight( i, 0f );
+			if( _debugRandomizer )
+			{
+				InvokeRepeating( "RandomizeBuddy", 0.3f, 0.3f );
+			}
+			else
+			{
+				CancelInvoke( "RandomizeBuddy");
+			}
+
+			_prevDebugRandomizer = _debugRandomizer;
 		}
```

Comment:
Is this just to give an inspector interface for triggering randomization over time? There's probably a better way to do that.